xicam/gui/bluesky/utils.py

```diff
@@ -6,77 +6,6 @@
 from traitlets import HasTraits, TraitType
 from traitlets.config import Configurable
 
-
```

Comment:
This MoveableTabWidget looks like it would still be a useful thing. Can we take a look and maybe pull it into `xicam.gui.widgets`?

Reply:
It's based on code I found on StackOverflow (linked in a comment, I think) and I found it to be pretty fragile. Dragging and dropping didn't always work, and it's unclear why. It looks like @klauer has something better (I'd have to go back through Slack to find it...) and it might be worth looking at that instead.

Reply:
My pure Python version of [Qt-Advanced-Docking-System](https://github.com/githubuser0xFFFF/Qt-Advanced-Docking-System), which I named [qtpydocking](https://github.com/klauer/qtpydocking), is deprecated. 

However, @hhslepicka is very close to having official bindings accepted at conda-forge. ref: https://github.com/conda-forge/staged-recipes/pull/10376